pkg/kubelet/dockertools/manager.go

```diff
@@ -249,6 +249,13 @@ func NewDockerManager(
 	}
 	dm.containerGC = NewContainerGC(client, podGetter, containerLogsDir)
 
+	dm.versionCache = cache.NewObjectCache(
+		func() (interface{}, error) {
+			return dm.getVersionInfo()
+		},
+		60*time.Second,
```

Comment:
And you may want to set this to 0 or use a fake expiration cache in fake manager.